src/Parsers/ExpressionListParsers.cpp

```diff
@@ -778,14 +776,13 @@ class Layer
     int state = 0;
 };
 
-
 struct ParserExpressionImpl
 {
-    static std::vector<std::pair<const char *, Operator>> operators_table;
-    static std::vector<std::pair<const char *, Operator>> unary_operators_table;
-    static const char * overlapping_operators_to_skip[];
+    static const std::vector<std::pair<std::string_view, Operator>> operators_table;
+    static const std::vector<std::pair<std::string_view, Operator>> unary_operators_table;
+    static const std::array<std::string_view, 1> overlapping_operators_to_skip;
```

Comment:
(Just FYI and for future changes: C++17 CTAD allows to omit the template arguments, i.e. `static const std::array arr = {...}`. Here, the deduced type would be `const char *` which isn't what we want, so we'd have to explicitly create std::string_view, e.g. by using the `sv` suffix: `= {"IN PARTITION"sv}` ... not really beautiful in this particular case, so template arguments are okay.

Reply:
That's pretty handy actually. Thank you very much!